spec/granite/columns/primary_key_spec.cr

```diff
@@ -11,10 +11,6 @@ describe "#new" do
 end
 
 describe "#new(primary_key: value)" do
-  it "ignores the value in default" do
-    Parent.new(id: 1_i64).id.should eq(nil)
-  end
```

Comment:
This wouldn't compile anymore assuming the user doesn't include `id` within the initializer (which wouldn't make sense since its auto increment anyway).